apps/site/assets/ts/schedule/components/schedule-finder/TableRow.tsx

```diff
@@ -55,28 +55,33 @@ const BusTableRow = ({
   journey: Journey;
   anySchoolTrips: boolean;
   isSchoolTrip: boolean;
-}): ReactElement<HTMLElement> => (
-  <>
-    {anySchoolTrips && (
-      <td className="schedule-table__td--tiny">
-        {isSchoolTrip && <strong>S</strong>}
+}): ReactElement<HTMLElement> => {
+  const backgroundClass = journey.route.name.startsWith("SL")
+    ? "u-bg--silver-line"
+    : "u-bg--bus";
+  const busPillClasses = `c-icon__bus-pill--small schedule-table__scheduled-bus-pill ${backgroundClass}`;
```

Comment:
This was a bit confusing for me to follow since these lines only become relevant within the `journey.route.type === 3` branch, and otherwise go unused. Perhaps a better way to extract this would be a separate component (it could just be defined in this file) that contains both this logic and the route-type conditional? e.g.

```jsx
<td className="schedule-table__td">
  <RouteIcon route={journey.route} />
  {breakTextAtSlash(journey.trip.headsign)}
</td>
```

Come to think of it, we already have a `RoutePill` component used elsewhere in `projects` — it doesn't look _identical_ to this logic, but I wonder if there's enough overlap to justify a refactor.